### PR TITLE
Run CKE as a normal process

### PR DESCRIPTION
**Pull request by ymmt2005**
This PR makes CKE run without a container and removes dependency on CKE from `go.mod`.
This change is necessary to enable CKE's graceful node reboot feature.

### CODE REVIEW

File: worker/cke.go [hunk 1]
```diff
@@ -26,14 +26,6 @@ func (o *operator) UpdateCKE(ctx context.Context, req *neco.UpdateRequest) error
...
 	if need {
-		err = cke.InstallToolsCKE(ctx)
-		if err != nil {
-			return err
-		}
-		err = cke.InstallBashCompletion(ctx)
-		if err != nil {
-			return err
-		}
 		err = o.storage.RecordContainerTag(ctx, o.mylrn, "cke")
```
Comment (ymmt2005) [inline worker/cke.go:29]: CKE is no longer installed as a container.

File: debian/DEBIAN/postinst [hunk 2]
```diff
@@ -1,9 +1,12 @@
 #!/bin/sh -e
 
-SERVICES="node-exporter neco-updater neco-worker sabakan-state-setter ingress-watcher"
+SERVICES="node-exporter neco-updater neco-worker sabakan-state-setter ingress-watcher cke"
 TIMERS="rkt-gc kill-old-login-sessions backup-cke-etcd"
 
 configure() {
+    # to upgrade from container version of CKE
+    rm -f /etc/systemd/system/cke.service
```
Comment (ymmt2005) [inline debian/DEBIAN/postinst:8]: @yokaze 
We need to stop containerized CKE in the previous installation, don't we?
```
systemctl stop cke.service || true
```

File: dctest/upgrade.go [hunk 3]
```diff
@@ -407,6 +413,27 @@ func checkVersionByDocker(address, name, image string) error {
 	return nil
 }
 
+func checkVersionOfCKE(host, image string) error {
```
Comment (ymmt2005) [inline dctest/upgrade.go:416]: I don't think this is necessary because now CKE is installed as a part of the Debian package.

**Review by yokaze — APPROVED** [review]
Approved by @ymmt2005

**Pull request merged**